server/db/seed.js

```diff
@@ -0,0 +1,101 @@
+const Promise = require("bluebird");
+const {
+  Address,
+  Category,
+  LineItem,
+  Order,
+  Product,
+  Review,
+  User
+} = require('./models');
+console.log(Category)
```

Comment:
Is this console.log for personal testing or do we want to keep it in our code for some more clarity?

Reply:
Mimi good catch! I had that earlier just for myself because I was getting a bug but it can be deleted now